pipelines/fast-test.yml

```diff
@@ -87,6 +87,10 @@ stages:
     variables:
```

Comment:
If NAS UI will be broken in this update, we should remove it from our code base. But I think it's okay to do it after the new NAS visualization is merged.

Reply:
I have not tested. Don't want to touch it...